node-fs-writefile/cp.js

```diff
@@ -0,0 +1,16 @@
+import { writeFile, readFile } from 'node:fs/promises';
+
+const sourceFile = process.argv[2];
+const destination = process.argv[3];
+
+async function processFiles() {
+  try {
+    const content = await readFile(sourceFile, 'utf8');
+    await writeFile(destination, content + '\n');
```

Comment:
When making an exact duplicate of a file, it's best not to add additional character.